public/v4/apps/forge_minecraft.yml

```diff
@@ -0,0 +1,98 @@
+captainVersion: 4
+services:
+  $$cap_appname:
+    image: itzg/minecraft-server:latest
+    environment:
+      EULA: "true"
+      TYPE: "FORGE"
+      VERSION: "$$cap_minecraft_version"
+      FORGE_VERSION: "$$cap_forge_version"
+      MAX_PLAYERS: "$$cap_max_players"
+      MODE: "$$cap_game_mode"
+      MAX_MEMORY: "$$cap_max_ramG"
+      SERVER_NAME: "$$cap_display_name"
+      MOTD: "$$cap_display_description"
+      ONLINE_MODE: "$$cap_online_mode"
+      RCON_PASSWORD: "$$cap_rcon_password"
+    ports:
+      - "$$cap_mc_port:25565"
+    volumes:
+      - $$cap_appname-data:/data
+    stdin_open: true
+    tty: true
+    restart: unless-stopped
+    caproverExtra:
+      notExposeAsWebApp: 'true'
+  $$cap_appname-web:
+    image: itzg/rcon
```

Comment:
Same with latest tag here. Not specifying tag is same as `latest`

Reply:
I added `multiarch` because it's the only one that will work on multi archs.